data/sql/updates/pending_db_world/rev_1667668503070084000.sql

```diff
@@ -0,0 +1,3 @@
+--
+DELETE FROM `gameobject_queststarter` WHERE (`quest` = 2933) AND (`id` IN (142703, 142704, 142705, 142706, 142707, 142708, 142713, 142714));
+INSERT INTO `gameobject_queststarter` (`id`, `quest`) VALUES (142703, 2933), (142704, 2933), (142705, 2933), (142706, 2933), (142707, 2933), (142708, 2933), (142713, 2933), (142714, 2933);
```

Comment:
so 142702 is what we have in Azerothcore currently that starts the quest.
Venom bottles to start the quest is 12 in total.
we have 13 in total in the gameobject_template when you search Venom Bottle.
![image](https://user-images.githubusercontent.com/16887899/200188158-e14f2d56-9f5e-4e1d-972b-e6afb9365dab.png)

This PR will bring the total amount of Venom bottles to start the quest to 9 in total.
What are we missing?

Reply:
Venom bottles 142708 through 142711 (4 total) dont currently have any spawns

Reply:
add what u feel like is needed to add to the completeness of the PR and to resolve the issue if possible.